factory/cluster.go

```diff
@@ -71,14 +71,14 @@ func DestroyCluster(g GinkgoTInterface) (string, error) {
 		return "", shared.ReturnLogError("error getting config: %w", err)
 	}
 
-	varDir, err = filepath.Abs(shared.BasePath() + fmt.Sprintf("/distros-test-framework/config/%s.tfvars", cfg.Product))
+	tfDir, err := filepath.Abs(shared.BasePath() + fmt.Sprintf("/distros-test-framework/modules/%s", cfg.Product))
```

Comment:
why getting from modules? we should maintain the pattern and best practice to load config from config. not from anywhere else.

Reply:
Sorry I am not getting the request. I am using modules path for tf dir

Reply:
sorry, i see that wrong